JellyfinPlayer tvOS/VideoPlayer/MediaInfoView.swift

```diff
@@ -0,0 +1,117 @@
+        contentView.view.bottomAnchor.constraint(equalTo: view.bottomAnchor).isActive = true
+        contentView.view.leftAnchor.constraint(equalTo: view.leftAnchor).isActive = true
+        contentView.view.rightAnchor.constraint(equalTo: view.rightAnchor).isActive = true
+        
+    }
+}
+
+struct MediaInfoView: View {
+    @State var item : BaseItemDto? = nil
+    
+    var body: some View {
+        if let item = item {
+            HStack {
+                VStack {
+                    ImageView(src: item.type == "Episode" ? item.getSeriesPrimaryImage(maxWidth: 200) : item.getPrimaryImage(maxWidth: 200), bh: item.type == "Episode" ? item.getSeriesPrimaryImageBlurHash() : item.getPrimaryImageBlurHash())                    .frame(width: 200, height: 300)
+                        .cornerRadius(10)
+                    Spacer()
+                }
+                .padding(.leading, 200)
+                    
+                    VStack(alignment: .leading) {
+                        if item.type == "Episode" {
+                            Text(item.seriesName!)
+                                .font(.title3)
+                            
+                            Text("S\(item.parentIndexNumber ?? 0):E\(item.indexNumber ?? 0) • \(item.name!)")
+                                .font(.headline)
+                                .foregroundColor(.secondary)
+                        }
+                        else
+                        {
+                            Text(item.name!)
+                                .font(.title3)
+                        }
+                        
+                        HStack(spacing: 10) {
+                            Text(String(item.productionYear!))
+                            Text("•")
+                            
+                            Text(formatRunningtime())
+                            
+                            if item.officialRating != nil {
+                                Text("•")
+                                
+                                Text("\(item.officialRating!)").font(.subheadline)
+                                    .fontWeight(.semibold)
+                                    .padding(EdgeInsets(top: 1, leading: 4, bottom: 1, trailing: 4))
+                                    .overlay(RoundedRectangle(cornerRadius: 2)
+                                                .stroke(Color.secondary, lineWidth: 1))
+                                
+                            }
+                        }
+                        .padding(.top)
+                        .foregroundColor(.secondary)
+                        
+                        
+                        Text(item.overview!)
+                            .padding([.top, .trailing])
+                            .foregroundColor(.secondary)
+                        
+                        Spacer()
+                    }
+                
+                
+                Spacer()
```

Comment:
Why are there `Spacer` here?
If you erase it, the center will be set.

https://github.com/jellyfin/SwiftFin/pull/82/files#r655937269

Reply:
and remove constant padding `.padding(.leading, 200)`

Reply:
Yeah currently fixing up the media info view. It seems that the native is not exactly centered.

Native player:
![Screen Shot 2021-06-22 at 5 48 33 pm](https://user-images.githubusercontent.com/47413006/122890834-64325d00-d387-11eb-9f7f-2917bd55b6ee.png)

Updated player:
![Simulator Screen Shot - Apple TV 4K (at 1080p) (2nd generation) - 2021-06-22 at 18 26 49](https://user-images.githubusercontent.com/47413006/122890971-7f9d6800-d387-11eb-92fb-a16067e39dbc.png)